ui/00_message/enemy/enemy_name.xml

```diff
@@ -1472,78 +1472,97 @@
   <message>
     <key>ENEMY_NAME_314</key>
     <original>グリムゴブリン</original>
+    <translation>Grimm Goblin</translation>
```

Comment:
Why the double m? Is there official English romanization of these enemy names somewhere? Otherwise it should be "Grim".